builtin/mainmenu/tab_credits.lua

```diff
@@ -97,6 +78,28 @@ local previous_contributors = {
 	"Jeija <jeija@mesecons.net> [HTTP, particles]",
 	"bigfoot547 [CSM]",
 	"Rogier <rogier777@gmail.com> [Fixes]",
+	"numberZero [Audiovisuals: meshgen]",
+	"stujones11 [Android UX improvements]",
+	"red-001 <red-001@outlook.ie> [CSM & Menu fixes]",
+	"Dániel Juhász (juhdanad) <juhdanad@gmail.com> [Audiovisuals: lighting]",
+	"Vincent Glize (Dumbeldor) [Cleanups, CSM APIs]",
+	"Ben Deutsch [Rendering, Fixes, SQLite auth]",
+	"DTA7 [Fixes, mute key]",
+	"Thomas-S [Disconnected, Formspecs]",
+	"Raymoo [Tool Capabilities]",
+	"noob3167 [Fixes]",
+	"adelcoding1 [Formspecs]",
+	"Rui [Sound Pitch]",
+	"Esteban (EXio4) [Cleanups]",
+	"Vaughan Lapsley (vlapsley) [Carpathian mapgen]",
+	"CoderForTheBetter [Add set_rotation]",
+	"Gaël-de-Sailly [Mapgen, pitch fly]",
+	"zeuner [Docs, Fixes]",
+	"ThomasMonroe314 (tre) [Fixes]",
+	"Rob Blanckaert (basicer) [Fixes]",
+	"Jozef Behran (osjc) [Fixes]",
+	"Pedro Gimeno (pgimeno) [Fixes]",
+	"lisacvuk [Fixes]",
```

Comment:
that's a lot of `[Fixes]`
is there some policy when people are removed from this list?

Reply:
I'm not sure how best to quantify this! Previous contributors should be people who have made huge contributions to Minetest, or recent big contributions

Reply:
Perhaps you could do something like a button:
### Core Developers
rubenwardy
sfan5
ect

### Contributors
[See full list]--Button

Reply:
At the very least contributors without commits in this or the previous release should not be in this list <b>if</b> their only contribution is "Fixes" or a minor feature.